server/routes.js

```diff
@@ -138,7 +138,9 @@ module.exports = (app) => {
   app.get('/groups/tags', groups.getGroupTags); // List all unique tags on all groups
   app.get('/groups/:groupid', aZ.authorizeAccessToGroup({authIfPublic: true}), groups.getOne);
   app.get('/groups/:groupid/users', aZ.authorizeAccessToGroup({authIfPublic: true}), cache(60), groups.getUsers); // Get group users
+  app.get('/groups/:groupid/active_users', aZ.authorizeAccessToGroup({authIfPublic: true}), cache(60), groups.getActiveUsers); // Get group users
```

Comment:
How about adding a flag to the endpoint of line 140 instead of introducing new endpoint? Same comment 2 lines below